packages/api/src/services/auth.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Context } from "../context";
+import argon2 from "argon2";
+import { signJwt, verifyJwt } from "../utils/jwt";
+import { config } from "../config/config";
+
+export const signIn = async (ctx: Context, email: string, password: string) => {
+  const user = await ctx.prisma.user.findFirstOrThrow({
+    where: { email: email },
+  });
+  if (!user.password) throw new Error("Invalid password");
```

Comment:
For error message kindly use trpc error, so server will get propper HTTP code error
https://trpc.io/docs/server/error-handling#throwing-errors